dashboard/client/src/common/JobStatus.tsx

```diff
@@ -75,6 +76,9 @@ const useJobStatusIconStyles = makeStyles((theme) =>
     colorError: {
       color: theme.palette.error.main,
     },
+    colorStopped: {
+      color: "#757575",
```

Comment:
I should use theme.palette here, but I don't the specific color value. Do you recommend me adding it to theme.palette?

Reply:
no, this is fine. mui v4, which is what we use, does not support adding new palette values